vortex-alp/src/compress.rs

```diff
@@ -99,6 +100,7 @@ where
                 PrimitiveArray::from(exc_pos).into_array(),
                 PrimitiveArray::from(exc).into_array(),
                 len,
+                Scalar::Null(NullScalar::new()),
```

Comment:
You actually want a typed scalar here. Something like `Scalar::null(dtype)`, though I don't know if that function exists.